### PR TITLE
updated governance.md with info on roles and responsibilities

### PR DESCRIPTION
**Pull request by tracykteal**
Updated governance page with info on roles and responsibilities. Wait for approval from @lexnederbragt @ethanwhite @k8hertweck or @karenword to merge.

### CODE REVIEW

**Comment by k8hertweck** [comment]
I approve! Thanks for the additional explanation.

File: pages/governance.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ permalink: /governance/
 
 <a href="{{site.url}}/team/">Tracy Teal</a> is the Executive Director of the Carpentries, and <a href="{{site.url}}/team/">Erin Becker </a> is the Associate Director. 
 
-Governance of the Carpentries is undertaken by the newly formed <strong>Executive Council</strong>, to whom the Executive Director reports. The Executive Council replaces the former Steering Committees of Software and Data Carpentry. The Council comprises nine members, four elected by the community, and five appointed from the former Steering Committees 
-of Software and Data Carpentry. 
+Governance of the Carpentries is undertaken by the <strong>Executive Council</strong>, to whom the Executive Director reports. The Executive Council replaces the former Steering Committees of Software Carpentry and Data Carpentry. The Council comprises nine members, four elected by the community, and five appointed from the former Steering Committees of Software Carpentry and Data Carpentry.
```
Comment (rgaiacs) [inline pages/governance.md]: Can we use double star, `**`, instead of `<strong>`?

**Comment by weaverbel** [comment]
Am I good to merge this now @tracykteal ? Looks like a few people have weighed in.

**Comment by tracykteal** [comment]
Yes, good to merge, thanks.

File: pages/governance.md (hunk 1 above)
Comment (tracykteal) [reply pages/governance.md]: yes, I'll make that change

**Pull request merged**